django_mongodb/expressions.py

```diff
@@ -66,6 +68,10 @@ def expression_wrapper(self, compiler, connection):
     return self.expression.as_mql(compiler, connection)
 
 
+def extra_where(self, compiler, connection):  # noqa: ARG001
```

Comment:
The other patched objects in Django's `where.py` are overridden in `query.py`, so let's organize this there.